### DISCUSSION TITLE
Should we add an `examples` directory for debugging and testing out new features?

### DISCUSSION THREAD

**Discussion by bytrangle** [decaporg/decap-cms#6491] [Ideas]
AFAIK the files in `dev-test` is mostly for debugging an issue. They get changed with each merged PR as the PR authors try to test out their fixes or the new features being introduced. Sometimes `dev-test` files can become very project-specific.

How about we create a dedicated `examples` directory to:
- showcase how Netlify CMS can be used with different frameworks and environments
- use as additional tooling to test new features

Friendly ping @krider2010 and anyone interested in making the testing of Netlify CMS more like a breeze.

**Comment by krider2010** [comment]
This is a great idea! Open to folks contributing.
